script.js

```diff
@@ -0,0 +1,133 @@
+// hamburger menu
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  document.body.classList.toggle('no-scroll');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+  document.body.classList.remove('no-scroll');
+}));
+
+// title for speaker section
+const spTitle = document.getElementById('sp-title');
+spTitle.innerHTML = `
+  <div id="spTitle" class="FeTitle d-flex flex-d-column">
+                <h3 class="spTitle font">Featured Speakers</h3>
+                <div class="line"></div>  
+            </div>
+  `;
+// get all speakers in one array
+const allSpeakers = [
+  {
+    id: '1',
+    name: 'Mohsin Ali',
+    quali: 'B.Se in Agriculture from Punjab University',
+    extra: 'Also got a diploma in Agriculture',
+    pic: './img/profiles/person1.jpg',
+  },
+  {
+    id: '2',
+    name: 'Muhammad Iqbal',
+    quali: 'B.Se in Agriculture from Punjab University',
+    extra: 'Also got a diploma in Agriculture',
+    pic: './img/profiles/person2.jpeg',
+  },
+  {
+    id: '3',
+    name: 'Robina Parveen',
+    quali: 'B.Se in Agriculture from Punjab University',
+    extra: 'Also got a diploma in Agriculture',
+    pic: './img/profiles/person3.jpeg',
+  },
+  {
+    id: '4',
+    name: 'Abid Ali',
+    quali: 'B.Se in Agriculture from Punjab University',
+    extra: 'Also got a diploma in Agriculture',
+    pic: './img/profiles/person4.jpeg',
+  },
+  {
+    id: '5',
+    name: 'Zeba Khan',
+    quali: 'B.Se in Agriculture from Punjab University',
+    extra: 'Also got a diploma in Agriculture',
+    pic: './img/profiles/person5.jpeg',
+  },
+  {
+    id: '6',
+    name: 'Abida Tufail',
+    quali: 'B.Se in Agriculture from Punjab University',
+    extra: 'Also got a diploma in Agriculture',
+    pic: './img/profiles/person6.jpeg',
+  },
+];
+
+// render all speakers in speaker container
+const speakerContainer = document.getElementById('allSpeakers');
+
+allSpeakers.map((speaker) => {
+  speakerContainer.innerHTML += `
+  <div id="allSpeakers" class="d-flex"> 
+              <div id="${speaker.id}" class="speakerCard d-flex">
+                <div class="checker">
+                    <img src="${speaker.pic}" alt="SpeakerImage">
+                </div>
+                <div class="spDetail">
+                  <h3 class="spName font">${speaker.name}</h3>
+                  <p class="spDesc font">${speaker.quali}</p>
+                  <div class="spline"></div>
+                  <p class="spStudy font">${speaker.extra}</p>
+                </div>
+             </div>
```

Comment:
- [ ] Please let's make sure that the speaker's section has only two columns and 3 rows as we have in the original design.

### Your design :point_down: 
![Screenshot from 2023-03-15 21-08-09](https://user-images.githubusercontent.com/89228743/225445041-f705b37a-8d58-4e87-9e4a-9c183cec6060.png)

### The original design :point_down: 
![Screenshot from 2023-03-15 21-13-53](https://user-images.githubusercontent.com/89228743/225445240-f60606f2-1845-4371-ab5e-4ec8e628822d.png)